MAUI/Segmented-Control/selection.md

```diff
@@ -0,0 +1,489 @@
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.SelectionIndicatorSettings = new SelectionIndicatorSettings()
+        {
+            Background = Colors.Blue,
+        };
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Customize Selected Segment Background of Each Segment Item
+
+You can change the selected segment background of the each segment item by using `SelectedSegmentBackground` property of `SfSegmentItem`.
+
+{% tabs %}
+{% highlight C# tabtitle="MainPage.xaml.cs"%}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        List<SfSegmentItem> itemsCollection = new List<SfSegmentItem>
+            {
+                new SfSegmentItem() {Text="Day", SelectedSegmentBackground = Colors.LightBlue},
+                new SfSegmentItem() {Text="Week", SelectedSegmentBackground = Colors.Blue},
+                new SfSegmentItem() {Text="Month", SelectedSegmentBackground = Colors.SkyBlue},
+                new SfSegmentItem() {Text="Year", SelectedSegmentBackground = Colors.DarkBlue},
+            };
+        segmentedControl.ItemsSource = itemsCollection;
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customize Selected Segment Text Color
+
+#### Customize Selected Segment Text Color of Segmented Control
+You can customize the selected segment text color, by using `TextColor` property of `SelectionIndicatorSettings`.
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage   
+    xmlns:buttons="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
+    <buttons:SfSegmentedControl x:Name="segmentedControl">
+        <buttons:SfSegmentedControl.SelectionIndicatorSettings>
+            <buttons:SelectionIndicatorSettings 
+                TextColor="Red"/>
+        </buttons:SfSegmentedControl.SelectionIndicatorSettings>
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.SelectionIndicatorSettings = new SelectionIndicatorSettings()
+        {
+            TextColor = Colors.Red,
+        };
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Customize Selected Segment Text Color of Each Segment Item
+
+You can change the selected segment text color of the each segment item by using `SelectedSegmentTextColor` property of `SfSegmentItem`.
```

Comment:
Change the selected segment text color of each segment item using the `SelectedSegmentTextColor` property of `SfSegmentItem`.

Reply:
content changed